Design/DesignDoc.md

```diff
@@ -0,0 +1,220 @@
+# Design Document
+
+## _AmazonBudgetService_ Design
+
+## 1. Problem Statement
+
+Amazon Elite Budget Service is a highly advanced budgeting app that will enable users to have more control over their money/expenditures.
+Users can currently view their expenses by visiting the associated website/app for their card provider. However, users have requested that their expenditures and budget plans should be presented in a more streamlined, informative manner.
+
+This design document describes the main use cases and functionality of Amazon Elite Budget Service, a new, native AWS service that will provide users a cleaner, more informative means to view expenditures and budget goals.
```

Comment:
In this context I wouldn't describe this as an AWS service - that reads like it's something that I can use within AWS (e.g. like Lambda or CloudFormation). You're building a service/app that runs _on_ AWS, but that's a level of detail that probably belongs more in the technical details below.